src/napari_imagej/utilities/_module_utils.py

```diff
@@ -411,6 +412,20 @@ def _get_postprocessors():
     return postprocessors
 
 
+prog_bars = {}
+
+
+def _init_progress(module: "jc.Module"):
+    prog_bars[module] = progress(desc=str(module.getInfo().getTitle()), total=3)
+
+
+def _update_progress(module: "jc.Module"):
+    if pbr := prog_bars.get(module):
+        pbr.update()
+        if pbr.total == pbr.n:
+            pbr.close()
```

Comment:
Consider encapsulating this within some container object/module - it will get easily lost here.

Reply:
Done in [fbc313c](https://github.com/imagej/napari-imagej/pull/200/commits/fbc313cff4d7526cf4c74cdf61e302f289714e15)